docs/diagrams/GoToUiClassDiagram.puml

```diff
@@ -0,0 +1,20 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor UI_COLOR_T4
+skinparam classBackgroundColor UI_COLOR
+
+
+Class MainWindow
+
+MainWindow *-down-> TravelPlanObjectListPanel
+MainWindow *-down-> TravelPlanPanel
+MainWindow *-down-> TravelPlannerPanel
+
+TravelPlanPanel -down-> NameCard
+
+TravelPlannerPanel -down-> ActivityCard
+TravelPlannerPanel -down-> AccommodationCard
+TravelPlannerPanel -down-> FriendCard
```

Comment:
These 3 should be TravelPlanObjectListPanel

Reply:
Oya ok i changed ps